src/app/assets/icons/QuestionmarkIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export default function QuestionmarkIcon(): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="25"
+      height="26"
+      viewBox="0 0 25 26"
+    >
+      <g transform="translate(-184 -237)">
+        <g
+          transform="translate(184 238)"
```

Comment:
Both `g` `transform` could be removed (sum is 0).
If a `g` is empty, it's safe to remove it.

Reply:
I tried it and it is not working. Think that is because there is another g and transform inside the outer g. This second g and transform relates to the text that lays within the ball.